### PR TITLE
rat cubes

### PR DESCRIPTION
**Pull request by fighterslam**
## About The Pull Request
Adds Rat Cubes to the game, orderable from cargo.
These rats cannot have diseases and thus are useless for virology

## Why It's Good For The Game
You can now acquire rats. Rats for the kitchen, rats for your favorite coworker, rats for your least favorite coworker.. Rats!

## Changelog
:cl: fighterslam
add: Added rat cubes, orderable from Cargo.
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
Lizards may now eat ratt

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -29,6 +29,7 @@
 	var/list/ratdisease = list()
 	can_be_held = TRUE
 	worn_slot_flags = ITEM_SLOT_HEAD
+	var/clean = FALSE
```
Comment (Archanial) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]:
```suggestion
	var/sterile = FALSE 
```

**Review by Archanial — CHANGES_REQUESTED** [review]

**Comment by St0rmC4st3r** [comment]
I think you should do something so that bounties could not be cheesed with this. Make it require wild rats from their natural habitat?

**Comment by Dingo-Dongler** [comment]
Why would you have the flavour text about the virologist running out of samples if they don't have any diseases

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 2]
```diff
@@ -40,7 +41,7 @@
 	icon_living = "mouse_[body_color]"
 	icon_dead = "mouse_[body_color]_dead"
 	held_state = "mouse_[body_color]"
-	if(prob(75))
+	if(prob(75) && !clean)
```
Comment (BrumalLake) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]:
```suggestion
	if(prob(75) && !sterile)
```

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 3]
```diff
@@ -149,3 +150,8 @@
 
 /obj/item/reagent_containers/food/snacks/deadmouse/on_grind()
 	reagents.clear_reagents()
+
+/mob/living/simple_animal/mouse/sterile
+	name = "lab mouse"
+	desc = "It's a skittish, clean-looking little rodent."
+	clean = TRUE
```
Comment (BrumalLake) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]:
```suggestion
	sterile = TRUE
```

**Review by BrumalLake — COMMENTED** [review]
changed a variable name without changing its instances?

**Comment by Boxlta** [comment]
RAT CUBE RAT CUBE RAT CUBE

**Comment by RyunaTW** [comment]
RATS RATS RATS RATS HAVE IT SPAWN A RAT NAME LOUIE RANDOMLY LIKE LISA CORGI RATS RATS RATS RATS

**Comment by fighterslam** [comment]
not ready for review quite yet, I need to re-make the rat-cube-box sprite before this is ready

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 4]
```diff
@@ -40,7 +41,7 @@
 	icon_living = "mouse_[body_color]"
 	icon_dead = "mouse_[body_color]_dead"
 	held_state = "mouse_[body_color]"
-	if(prob(75))
+	if(prob(75)) && !sterile)
```
Comment (BrumalLake) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:44]:
```suggestion
	if(!sterile && prob(75))
```
unpaired and it'd be better to check if it's sterile before computing rand

**Comment by ike709** [comment]
Stale

**Comment by Archanial** [comment]
Stale

**Pull request closed without merging**